mediorum/server/server.go

```diff
@@ -311,6 +311,17 @@ func New(config MediorumConfig) (*MediorumServer, error) {
 	routes.GET("/delist_status/user/:userId", ss.serveUserDelistStatus)
 	routes.POST("/delist_status/insert", ss.serveInsertDelistStatus, ss.requireBodySignedByOwner)
 
+	// -------------------
+	// healthz
+	healthzUrl, err := url.Parse("http://healthz")
+	if err != nil {
+		log.Fatal("Invalid healthz URL: ", err)
+	}
+	healthz := routes.Group("/healthz")
+	healthz.Use(middleware.Proxy(middleware.NewRandomBalancer([]*middleware.ProxyTarget{
```

Comment:
kind of a nit and i see where this snippet comes from on official docs, but a load balancer with a single target makes more sense to be implemented like:

```go
proxyTarget := httputil.NewSingleHostReverseProxy(healthzURL)
echoServer.Any("/healthz", echo.WrapHandler(proxyTarget))
```

can't remember off the top of my head but it might need to be `echoServer.Any("healthz*")` if subroutes aren't being redirected

Reply:
Excellent, thanks for the pointer